README.md

```diff
@@ -118,6 +118,7 @@ OPTIONS:
 | F634 | IfTuple | If test is a tuple, which is always `True` |
 | F704 | YieldOutsideFunction | a `yield` or `yield from` statement outside of a function/method |
 | F706 | ReturnOutsideFunction | a `return` statement outside of a function/method |
+| F707 | DefaultExceptNotLast | an `except:` block as not the last exception handler |
```

Comment:
For the future: if you add `CheckKind::DefaultExceptNotLast` to the list in `examples/generate_rules_table.rs`, then run `cargo run --example generate_rules_table`, it'll print out the Markdown table, which you can then paste in here.

Reply:
Got it, I missed that :)

Reply:
Np, hopefully makes your life easier! (`examples` is a weird home for that script but I didn't want to introduce multiple binaries.)